docs/modules/plugins/pages/plugin-accessibility.adoc

```diff
@@ -0,0 +1,47 @@
+= Accessibility Plugin
+
+The plugin provides functionality to perform accessbility validations
+For more info please see https://squizlabs.github.io/HTML_CodeSniffer/[HTML Code Sniffer]
+Suppoerted checks against following standads: https://squizlabs.github.io/HTML_CodeSniffer/Standards/WCAG2/[WCAG2] and https://squizlabs.github.io/HTML_CodeSniffer/Standards/Section508/[Section 508]
+
+== Installation
+
+.build.gradle
+[source,gradle,subs="attributes+"]
+----
+implementation(group: 'org.vividus', name: 'vividus-plugin-accessibility', version: '{current-version}')
+----
+
+== Steps
+
+=== Validate accessibility
+
+[source,gherkin]
+----
+Then all page elements are accessible:$options
+----
+
+* `$options` - The accessibility validation options. Where:
+** `standard` - [*Mandatory*] The accessibility stanadard to verify against. One of `WCAG2A`, `WCAG2AA`, `WCAG2AAA`, `Section 508`
+** `level` - [*Mandatory*] The violation level used to check. One of `ERROR` - only errors will be validated, `WARNING` - error and warning will be validated, `NOTICE` - all the levels will be valiadted
+** `locators` - The comma-separeted xref:vividus:plugins:plugin-web-app.adoc#_locator[locators] of elemements to verify
+** `elementsToIgnore` - The comma-separeted xref:vividus:plugins:plugin-web-app.adoc#_locator[locators] of elemements to ignore
+** `violationsToIgnore` - The vilations codes which should be ignored
+** `violationsToCheck` - The violations codes which should be checked
+
+[WARNING]
+If violation code will be set as checked and ignored then it will be checked and not ignored
+
+[TIP]
+The step based on contextual approach, and when necessary could be used to validated accessbility on the context element only
```

Comment:
`The step based on the contextual approach and when it's necessary it could be used to validate the accessibility of the context element only.`